linux/datadog_wrapper

```diff
@@ -90,7 +92,8 @@ setUpBinary() {
     if [ "$BINARY" = "dogstatsd" ]; then BINARY="dogstatsd start"; fi
 
     echo "Starting $BINARY"
-    ./"$BINARY" &
+    eval "${DD_DIR}/${BINARY} &"
```

Comment:
This change is confusing to me. How was it working before if it now needs the `$DD_DIR` path? Have you tested this with all the runtimes?

Reply:
I switched to the absolute path while debugging an issue. I figured it was better to eliminate relative paths moving forward.